client/src/components/WorkflowInvocationState/WorkflowInvocationState.vue

```diff
@@ -333,24 +333,36 @@ async function onCancel() {
             <BTab
                 v-if="!props.isSubworkflow"
                 title-item-class="invocation-report-tab"
-                :disabled="reportTabDisabled"
+                :disabled="tabsDisabled"
                 :lazy="reportLazy"
                 :active.sync="reportActive">
                 <template v-slot:title>
                     <span>Report</span>
                     <BBadge
-                        v-if="reportTabDisabled"
+                        v-if="tabsDisabled"
                         v-b-tooltip.hover.noninteractive
-                        :title="disabledReportTooltip"
+                        :title="disabledTabTooltip"
                         variant="warning">
                         <FontAwesomeIcon :icon="faExclamation" />
                     </BBadge>
                 </template>
                 <InvocationReport v-if="invocationStateSuccess" :invocation-id="invocation.id" />
             </BTab>
-            <BTab title="Export" title-item-class="invocation-export-tab" lazy>
+            <BTab title-item-class="invocation-export-tab" :disabled="tabsDisabled" lazy>
+                <template v-slot:title>
+                    <span>Export</span>
+                    <BBadge
+                        v-if="tabsDisabled"
+                        v-b-tooltip.hover.noninteractive
+                        :title="disabledTabTooltip"
+                        variant="warning">
+                        <FontAwesomeIcon :icon="faExclamation" />
+                    </BBadge>
+                </template>
                 <div v-if="invocationAndJobTerminal">
-                    <WorkflowInvocationExportOptions :invocation-id="invocation.id" />
+                    <WorkflowInvocationExportOptions
+                        :invocation-id="invocation.id"
+                        :terminal="invocationAndJobTerminal" />
                 </div>
                 <BAlert v-else variant="info" show>
```

Comment:
oh that's weird, my change initially was 
```
<WorkflowInvocationMetrics :invocation-id="invocation.id" :terminal="invocationAndJobTerminal" />
```

but i think the rebase messed it up 🤯 🤔 😩

Reply:
Yup, the lines are off now :laughing: 
But I was referring to the `<LoadingSpan message="Waiting to complete invocation" />`. It is not needed anymore since you cannot access the tab when the invocation has not finished.